src/system/SystemPacketBuffer.h

```diff
@@ -142,6 +144,20 @@ class DLL_EXPORT PacketBuffer : private pbuf
 #endif // CHIP_SYSTEM_CONFIG_USE_LWIP
 
 public:
+    void DebugDump(const char * title) const
+    {
+#ifdef DEBUG
+        printf("DebugDump %s message DataLength=%d\n", title, DataLength());
```

Comment:
Should probably use logging, not printf?  I guess that's hard for the per-byte bits....

Reply:
addressed in latest